swarm-production/docker-stack.yml

```diff
@@ -89,6 +94,90 @@ services:
         delay: "10s"
         monitor: "5s"
 
+  queue:
+    image: 'wurstmeister/kafka:1.0.0'
+    environment:
+      KAFKA_ADVERTISED_PORT: 9092
+      KAFKA_ADVERTISED_HOST_NAME: queue
+      KAFKA_CREATE_TOPICS: 'backbeat-replication:1:1'
+      KAFKA_ZOOKEEPER_CONNECT: 'quorum:2181'
+    ports:
+      - 9092
+    networks:
+      - backend
+    volumes:
+      - "queue-journal:/kafka:rw"
```

Comment:
do you have to have a volume for the journal as well as a volume for zk?

Reply:
I would not encourage using the same volume for different services